apps/pipelines/nodes/nodes.py

```diff
@@ -143,6 +143,22 @@ class HistoryMixin(LLMResponseMixin):
             widget=Widgets.none,
         ),
     )
+    history_mode: PipelineChatHistoryModes = Field(
```

Comment:
Might be good to add a `title`. Something like "History management strategy".

Reply:
Thanks @SmittieC, good point about a default. There is a default set via the UI but it would make sense to have a default set here too.